CI/build-macos.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+##############################################################################
+# macOS build script
+##############################################################################
+#
+# This script contains all steps necessary to:
+#
+#   * Build OBS with all default plugins and dependencies
+#   * Create a macOS application bundle
+#   * Code-sign the macOS application-bundle
+#   * Package a macOS installation image
+#   * Notarize macOS application-bundle and/or installation image
+#
+# Parameters:
+#   -h, --help                     : Print usage help
+#   -q, --quiet                    : Suppress most build process output
+#   -v, --verbose                  : Enable more verbose build process output
+#   -d, --skip-dependency-checks   : Skip dependency checks (default: off)
+#   -b, --bundle                   : Create relocatable app bundle
+#                                    (default: off)
+#   -p, --package                  : Create distributable disk image
+#                                    (default: off)
+#   -c, --codesign                 : Codesign OBS and all libraries
+#                                    (default: ad-hoc only)
+#   -n, --notarize                 : Notarize OBS (default: off)
+#   --build-dir                    : Specify alternative build directory
+#                                    (default: build)"
+# Environment Variables (optional):
+#
+#   MACOS_DEPS_VERSION        : Pre-compiled macOS dependencies version
+#   MACOS_CEF_BUILD_VERSION   : Chromium Embedded Framework version
+#   VLC_VERISON               : VLC version
+#   SPARKLE_VERSION           : Sparke Framework version
+#
+##############################################################################
+
+# Halt on errors
+set -eE
+
+## SET UP ENVIRONMENT ##
+_RUN_OBS_BUILD_SCRIPT=TRUE
+PRODUCT_NAME="OBS-Studio"
+
+CHECKOUT_DIR="$(/usr/bin/git rev-parse --show-toplevel)"
+DEPS_BUILD_DIR="${CHECKOUT_DIR}/../obs-build-dependencies"
+source "${CHECKOUT_DIR}/CI/include/build_support.sh"
+source "${CHECKOUT_DIR}/CI/include/build_support_macos.sh"
+
+## DEPENDENCY INSTALLATION
```

Comment:
Prefer "INSTALL DEPENDENCIES" (verb object - imperative mood).

Some comments seem to be used as headers and end with `##`, though some, like this line, do not end with `##`.  Some are UPPERCASE while some are Sentence case.  I assume there is some intent behind the different formatting?

Reply:
Yeah `INSTALL DEPENDENCIES` it should be.